synapse/config/metrics.py

```diff
@@ -13,7 +13,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from ._base import Config
+from ._base import Config, ConfigError
+
+MISSING_SENTRY = (
+    """Missing sentry_sdk library. This is required for enable sentry
+    integration.
+
+    Install by running:
+        pip install sentry_sdk
```

Comment:
I don't think we should recommend this. It's only useful for people who don't know what they are doing, who are exactly the sort of people who are likely to run this as root and hose their whole system.

Reply:
(anyway, it's sentry-sdk)

Reply:
Hmm, true, though it is the same error message we use for other missing dependencies.

I do think we need to say *something*, even I wouldn't necessarily know the right way is to do `pip install matrix-synapse[sentry]` or whatever.

Reply:
we may need an "installing optional dependencies" section in INSTALL.md which can explain that it is different whether you are installing from source or a system package.

In the meantime I honestly think it's better to say nothing.